src/main/java/org/hydev/mcpm/client/interaction/ProgressBar.java

```diff
@@ -18,50 +34,61 @@ public class ProgressBar implements AutoCloseable
     public ProgressBar(ProgressBarTheme theme)
     {
         this.theme = theme;
-        this.init();
-    }
+        this.out = System.out;
+        this.cu = new ConsoleUtils(this.out);
+        this.activeBars = new ArrayList<>();
+        this.cols = AnsiConsole.getTerminalWidth();
```

Comment:
Gave this a test run on my local machine. I've had bad luck with accomplishing these kinds of things without going through ncurses, and it seems like what we have right now is broken with whatever IntelliJ configuration I have set up.

Seems like `AnsiConsole.getTerminalWidth()` is returning 0, which is causing ProgressRow#toString line 72 to pass a negative count to repeat.

<img width="1110" alt="Screenshot 2022-10-30 at 3 56 49 PM" src="https://user-images.githubusercontent.com/32211852/198898918-717da3e2-1d17-4472-81a7-ae6f3d607002.png">

I'll try to run outside of IntelliJ later :)

Reply:
Yea IntelliJ uses Gradle to run the project, and Gradle's output isn't a tty, and Xterm control sequences only support tty environments. So you can only test it using external terminals or manually ran using IntelliJ's built-in terminal outside of gradle. I've included instructions in the readme

Reply:
Thanks! Can you detect this so if the TA is tinkering they don't hit an exception 7 levels deep? DW if you're busy with other things, but if it's a 2 second fix I'd recommend tackling it!

Reply:
Okay just fixed this